packages/docusaurus/src/client/exports/BrowserOnly.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, {useEffect, useState} from 'react';
+
+const BrowserOnly = ({children}) => {
```

Comment:
This also works and is shorter IMO.

```
function BrowserOnly({children}) {
  return ExecutionEnvironment.canUseDOM && children != null && <>{children()}</>;
};
```

Reply:
Thanks, although this does not solve the issue of importing components using `window`.

For example, if I just import this component in the MDX file, then I will have an error during the building that the `window` is not defined.

```js
import {Replay} from 'vimond-replay';
```

Reply:
In that case, users will have to lazy load the component after the component has mounted. There's no way we can handle that with the suggested component here.